docs/documentation/release_notes/topics/26_5_0.adoc

```diff
@@ -38,4 +38,13 @@ This provides a more fine-grained way to manage features and eliminates the need
 
 The `feature-<name>` option takes precedence over both `features` and `features-disabled`.
 
-For more details, see the https://www.keycloak.org/server/features[Enabling and disabling features] guide.
\ No newline at end of file
+For more details, see the https://www.keycloak.org/server/features[Enabling and disabling features] guide.
+
+== MDC Logging feature (supported)
+
+The log_mdc feature has been promoted from Preview to Supported(Default) Status.
```

Comment:
We should refer to the feature using the same format that users specify it on the CLI, i.e. `log-mdc`.

I would also remove the use of capitals for the feature state and the reference to "Default".

How about:  "The log-mdc has been promoted from preview to fully supported."